_posts/2021-04-14-cockpit-242.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Cockpit 242
+author: marusak
+date: 2021-04-14
+tags: cockpit linux
+slug: cockpit-242
+category: release
+summary: Cockpit with split out Machines page and website refresh
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 242.
+
+
+## Groundwork for Snowpack support
+
+[Snowpack](https://www.snowpack.dev/) is a web project build system alternative to [webpack](https://webpack.js.org/). Previously, when using it to build a Cockpit page which used an [npm module](https://www.npmjs.com/) containing an `@`, such as [@patternfly](https://www.npmjs.com/package/@patternfly/react-core), the page [could not be loaded](https://github.com/cockpit-project/cockpit/issues/15288). This has been fixed.
+
+Note that there is not yet a fully working and supported example of a snowpack-built Cockpit project, just some [initial experimentation](https://piware.de/post/2021-02-05-snowpack/).
```

Comment:
```suggestion
Please be aware that there is not a fully working and supported example of a Snowpack-built Cockpit project at this time, just some [initial experimentation](https://piware.de/post/2021-02-05-snowpack/).
```

- Changed sentence intro (optional)
- Fixed case of Snowpack
- Replaced _yet_ with _at this time_ (in a different part of the sentence)